### ISSUE TITLE
Invalid operation name under python 3.6.0

### ISSUE THREAD

**Issue by AlanCristhian** [haypo/bytecode#17]
Hello, I tried a very simple snippet and I get this:

```
>>> from bytecode import Bytecode
>>> Bytecode.from_code((lambda: x).__code__)
Traceback (most recent call last):
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/instr.py", line 199, in _set
    opcode = _opcode.opmap[name]
KeyError: '<0>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/bytecode.py", line 117, in from_code
    concrete = _bytecode.ConcreteBytecode.from_code(code)
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/concrete.py", line 143, in from_code
    instr = ConcreteInstr.disassemble(lineno, code.co_code, offset)
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/concrete.py", line 91, in disassemble
    return cls(name, arg, lineno=lineno)
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/concrete.py", line 36, in __init__
    self._set(name, arg, lineno)
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/concrete.py", line 49, in _set
    super()._set(name, arg, lineno)
  File "/home/alancristhian/py360/lib/python3.6/site-packages/bytecode/instr.py", line 201, in _set
    raise ValueError("invalid operation name")
ValueError: invalid operation name
```

**Comment by MatthieuDartiailh** [comment]
I cannot reproduce on master what version are you running ?

**Comment by haypo** [comment]
Oh right, bytecode 0.4 doesn't support Python 3.6.

I just released bytecode 0.5 which adds Python 3.6 support, enjoy ;-)

**Comment by AlanCristhian** [comment]
Thanks! :)
